backend-schedule/src/main/java/eu/socialedge/hermes/backend/schedule/domain/api/ScheduleGenerationApi.java

```diff
@@ -17,51 +17,45 @@
 import eu.socialedge.hermes.backend.schedule.domain.BasicScheduleGenerator;
 import eu.socialedge.hermes.backend.schedule.domain.ScheduleSpecification;
 import eu.socialedge.hermes.backend.schedule.domain.repository.ScheduleRepository;
-import eu.socialedge.hermes.backend.transit.domain.Route;
-import eu.socialedge.hermes.backend.transit.domain.repository.RouteRepository;
+import eu.socialedge.hermes.backend.transit.domain.repository.LineRepository;
 import lombok.val;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.rest.webmvc.RepositoryRestController;
 import org.springframework.http.ResponseEntity;
+import org.springframework.util.AntPathMatcher;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.util.UriComponentsBuilder;
 
 import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
 
-import java.util.Optional;
-
 import static org.springframework.web.bind.annotation.RequestMethod.POST;
 
-@RestController
-@RequestMapping(path = "/schedules")
+@RepositoryRestController
```

Comment:
imgo ScheduleGenerationApi is an API for schedule generator, not schedule and its repository. Revert `@RestController` pls